docs/products/kafka/howto/governance.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Governance in Aiven for Apache Kafka®
+sidebar_label: Governance How-To
+limited: true
+---
+Governance in Aiven for Apache Kafka® helps manage your Aiven for Apache Kafka clusters securely and efficiently through structured policies, roles, and processes.
+
+import DocCardList from '@theme/DocCardList';
```

Comment:
There is just one sections within this category. I assume you created it coz you're planning to add more in the future.

Reply:
Yes!